dist/redux-modules.cjs.js

```diff
@@ -3571,14 +3555,22 @@ var taxonomies = function taxonomies(state) {
 
 var orderedTaxonomy = function orderedTaxonomy(state, taxonomy) {
   var orderBy = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : 'title';
+  var withoutArchived = arguments.length > 3 && arguments[3] !== undefined ? arguments[3] : false;
 
-  return taxonomies(state).get(taxonomy, Immutable__default.Map()).toOrderedMap().sort(function (a, b) {
+  var result = taxonomies(state).get(taxonomy, Immutable__default.Map()).toOrderedMap().sort(function (a, b) {
     if (Number.isInteger(a.get(orderBy))) {
       return a.get('id') - b.get('id');
     } else {
       return a.get(orderBy).localeCompare(b.get(orderBy));
     }
   });
+
+  if (withoutArchived) {
+    result = result.delete("16");
```

Comment:
Can you filter by result.archived instead?

Reply:
That's not the latest commit.

Reply:
Ah I see, can you squash your commits please